src/rust/cxx-integration-test/cxx-rust-integration-test.c++

```diff
@@ -33,9 +33,9 @@ KJ_TEST("err Result") {
   try {
     rust::test::result_error();
     KJ_FAIL_REQUIRE("exception is expected");
-  } catch (::rust::Error e) {
+  } catch (kj::Exception e) {
     // this is expected
```

Comment:
We often use `kj::getCaughtExceptionAsKj()`, will that work here also? If it should, can you include a test for it?

Reply:
https://github.com/cloudflare/workerd/pull/4674